GitHub上で自動テストを行うためワークフローファイルの追加

kitazawasyogo opened the pull request:
@nabemasamasa　@Takatatsu9

GitHub上で自動テストを行うためワークフローファイルの追加を行いました。
レビューお願いいたします。

kitazawasyogo committed:
GitHub上で自動テストを行うためワークフローファイルの追加

Takatatsu9 commented:
見たこと無いエラーだ・・・

Takatatsu9 commented:
ymlファイル最後の`Bowling.Test\bin\Debug\Bowling.Test.dll`の`bin`を`obj`に変えてみてもらっていいですか？

Takatatsu9 commented:
（変えてもだめかもしれない・・・。だめだったらまた`bin`に戻してもらう）

kitazawasyogo commented:
`Bowling.Test\bin\Debug\Bowling.Test.dll`をサクラエディタで開きましたが、どの部分の変更をすればいいのか理解できていません。教えていただけますでしょうか。

> binをobjに変えてみてもらっていいですか？

Takatatsu9 commented:
変えてと言ったのはymlファイルですよ。issueで追加したと言った`dotnet4.yml`以外に
ymlファイルは存在しないと思うんですけど。

とはいえ、変更いらないかもしれません。ファイルが不足してることで落ちてる可能性が高くなりました。

Takatatsu9 committed:
存在しないと言われているファイルを追加

Takatatsu9 commented:
動いたけどまた別のエラーだ・・・

nabemasamasa committed:
異なる.NET Frameworkを吸収してテスト実行してくれるらしいのでnunit-agentを追加

When the program needs to be run under a different framework version
from the one begin used by NUnit itself
https://docs.nunit.org/2.5.10/nunit-agent.html

Takatatsu9 commented on External-Bin/Net/2.0/nunit-agent.exe.config:
```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="utf-8"?>
```
> 異なる.NET Frameworkを吸収してテスト実行してくれるらしいのでnunit-agentを追加

👍

Takatatsu9 commented:
動いた！大丈夫そうなのでMergeします！
